HLTrigger/Configuration/python/HLT_75e33/modules/hltHpsSelectedPFTausTrackPt1MediumChargedIsolation_cfi.py

```diff
@@ -0,0 +1,16 @@
+import FWCore.ParameterSet.Config as cms
+
+hltHpsSelectedPFTausTrackPt1MediumChargedIsolation = cms.EDFilter( "PFTauSelector",
+    src = cms.InputTag( "hltHpsPFTauProducer" ),
+    cut = cms.string( "pt > 0" ),
+    discriminators = cms.VPSet( 
+      #cms.PSet(  discriminator = cms.InputTag( "hltHpsPFTauTrackPt1Discriminator" ),
+        #selectionCut = cms.double( 0.01 )
+      #),
```

Comment:
Still some commented code. Remove if possible.

Reply:
I kept it on purpose, since it should be probably there based on Run2 paths but currently that part gives an error. We should still understand it. Of course we can remove it and once we fix it, we can add it later. If you suggest to do so, I can do it.

Reply:
Since there is a reason to possibly keep it, I don't have a strong preference. I let you decide.

Reply:
ok, then I prefer to keep it just to be on the safe side that the information won't be lost.